playbooks/group_vars/webportals.yml

```diff
@@ -113,16 +113,24 @@ webportal_allowance:
 webportal_server_config_defaults:
   domain_name: "{{ ansible_host }}"
   portal_name: "{{ ansible_host }}"
-  portal_modules: ""
+  portal_modules: "m"
   sia_wallet_password: ""
   sia_api_port: 9980
+  sia_api_password: "{{ lookup('pipe','openssl rand -base64 32') }}"
```

Comment:
is this gonna work? sia_api_password should equal the one that skyd generates on the first run right?

Reply:
yup, skyd checks for an environment variable first. So even if it previously created one on disk, when it restarts it will start using the new env var.

Reply:
This will break nginx reading api password since it expects it to be stored on a disk in our setup.

https://github.com/SkynetLabs/skynet-webportal/blob/master/docker/nginx/libs/skynet/utils.lua

Reply:
Plan is to update nginx

Reply:
resolved in https://github.com/SkynetLabs/skynet-webportal/pull/1567